doctr/models/classification/vit/tensorflow.py

```diff
@@ -31,6 +31,23 @@
 }
 
 
+class ClassifierHead(layers.Layer, NestedObject):
+    """Classifier head for Vision Transformer
+
+    Args:
+        num_classes: number of output classes
+    """
+
+    def __init__(self, num_classes: int) -> None:
+        super().__init__()
+
+        self.head = layers.Dense(num_classes, kernel_initializer="he_normal")
+
+    def call(self, x: tf.Tensor) -> tf.Tensor:
+        # (batch_size, num_classes) cls token
+        return self.head(x[:, 0])
+
```

Comment:
same comment as the other PR :)

Reply:
@frgfm mh but we have to grab especially the cls_token vector :thinking:

Reply:
I think in this case it is fine as it is ( I agree that a class for only a linear layer is a bit overhead but so we have a clear split) and it is unified with TF implementation :rofl: